src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp

```diff
@@ -49,7 +49,10 @@ static void CreateInterpolateOp(Program& p, const std::shared_ptr<ngraph::op::v4
         }
     }
 
-    OPENVINO_ASSERT(!scales_constant || axes.size() == scales.size(), op->get_friendly_name(), " Incorrect axes and scales should be the same size");
+    bool correct_params = !scales_constant;
+    correct_params |= axes.size() == scales.size();
```

Comment:
It can fail if the axes are default and scales are provided with smaller size than data rank,
for example `[H, W, C]` data input, scales `[0.5, 0.6]` then default axes `[0, 1, 2]` so `axes.size() != scales.size()`
cc: @tomdol

Reply:
Is it applicable for Interpolate_v4? According to specs, user should specify axes input explicitly or `[0,...,rank(data) - 1]` will be used as default values. So, shouldn't user in your example specify scales with 3 values or specify axes with 2 explicitly?

Reply:
We are trying to clarify that:) There is some space for different interpretation of this restriction, but it was possible to create such v4::Interpolate op, what doesn't mean it was supported by plugins: https://github.com/openvinotoolkit/openvino/blob/8c6910043916d5aa6e5dd17b786eb095854eed62/src/core/tests/visitors/op/interpolate.cpp#L48-L64

Reply:
@mitruska  Oh, I see... It's a really tricky thing
Are you okay to merge this PR before this question will be solved? It fixes quite a lot of issues, but doesn't change the behavior dramatically. But if it will be decided to allow patterns like you provided, we can handle it in separate PR